### PR TITLE
Add helpers for sso config

### PR DESCRIPTION
**Pull request by maniackcrudelis**
## The problem

We have lost the documentation about sso config keys

## Solution

Add helpers to add directly such config keys.
And bring documentation back with those helpers.

## PR Status

Ready to be reviewed.

## How to test

...

## Validation

- [ ] Principle agreement 0/2 : 
- [ ] Quick review 0/1 : 
- [ ] Simple test 0/1 : 
- [ ] Deep review 0/1 : 

@YunoHost/apps

### CODE REVIEW

File: data/helpers.d/setting [hunk 1]
```diff
@@ -59,6 +59,98 @@ ynh_app_setting_delete() {
...
+#
+# usage: ynh_add_skipped_uris [--appid=app] --url=url1,url2 [--regex]
+# | arg: -a, --appid - the application id
+# | arg: -u, --url - the urls to add to the sso for this app
+# | arg: -r, --regex - Use the key 'skipped_regex' instead of 'skipped_uris'
+#
+# An URL set with 'skipped_uris' key will be totally ignored by the SSO,
+# which means that the access will be public and the logged-in user information will not be passed to the app.
+#
+# Requires YunoHost version ?.?.? or higher.
```
Comment (kay0u) [inline data/helpers.d/setting]: Will be merged in 3.6.0, so change "?"

File: data/helpers.d/setting [hunk 2]
```diff
@@ -59,6 +59,98 @@ ynh_app_setting_delete() {
...
+#
+# usage: ynh_add_unprotected_uris [--appid=app] --url=url1,url2 [--regex]
+# | arg: -a, --appid - the application id
+# | arg: -u, --url - the urls to add to the sso for this app
+# | arg: -r, --regex - Use the key 'unprotected_regex' instead of 'unprotected_uris'
+#
+# An URL set with unprotected_uris key will be accessible publicly, but if an user is logged in,
+# his information will be accessible (through HTTP headers) to the app.
+#
+# Requires YunoHost version ?.?.? or higher.
```
Comment (kay0u) [inline data/helpers.d/setting]: Same here

File: data/helpers.d/setting [hunk 3]
```diff
@@ -59,6 +59,98 @@ ynh_app_setting_delete() {
...
+# Add protected_uris urls into the config
+#
+# usage: ynh_add_protected_uris [--appid=app] --url=url1,url2 [--regex]
+# | arg: -a, --appid - the application id
+# | arg: -u, --url - the urls to add to the sso for this app
+# | arg: -r, --regex - Use the key 'protected_regex' instead of 'protected_uris'
+#
+# An URL set with protected_uris will be blocked by the SSO and accessible only to authenticated and authorized users.
+#
+# Requires YunoHost version ?.?.? or higher.
```
Comment (kay0u) [inline data/helpers.d/setting]: And here

File: data/helpers.d/setting (hunk 1 above)
Comment (maniackcrudelis) [reply data/helpers.d/setting]: Are you sure it's going to be ?
This should be done during the testing release.

And could be done already for the previous release.

Comment (kay0u) [reply data/helpers.d/setting]: @alexAubin add it to the milestone 3.6.x (but this comment is more a reminder for him)

I can't find these helpers O.o

**Pull request merged**